README.md

```diff
@@ -74,6 +74,63 @@ export const App = () => (
 );
 ```
 
+## A11y Example
+
+```tsx
+import * as React from 'react';
+
+import { SnapList, SnapItem } from 'react-snaplist-carousel';
+
+const MyItem = ({ children }) => (
+  <div style={{ width: '70vw', height: 200, background: '#cccccc' }}>
+    {children}
+  </div>
+);
+
+export const App = () => {
+  const snapList = useRef(null);
+  const lastSnapItem = useRef(null);
+  const goToSnapItem = useScroll({ ref: snapList });
+  return (
+    <SnapList
+      ref={snapList}
+      tabIndex={0} // so it can receive focus and can be scrolled with keyboard
+      role="region" // context for screen readers
+      aria-label="my awesome snaplist" // for screen readers to read out loud on focus
+    >
+      <SnapItem margin={{ left: '15px', right: '15px' }} snapAlign="center">
+        <MyItem>Item 0</MyItem>
+        <button
+          onClick={() => {
+            goToSnapItem(4);
+            lastSnapItem.current?.focus();
+          }}
+        >
+          go to last item
+        </button>
+      </SnapItem>
+      <SnapItem margin={{ left: '15px', right: '15px' }} snapAlign="center">
+        <MyItem>Item 1</MyItem>
+      </SnapItem>
+      <SnapItem margin={{ left: '15px', right: '15px' }} snapAlign="center">
+        <MyItem>Item 2</MyItem>
+      </SnapItem>
+      <SnapItem margin={{ left: '15px', right: '15px' }} snapAlign="center">
+        <MyItem>Item 3</MyItem>
+      </SnapItem>
+      <SnapItem
+        margin={{ left: '15px', right: '15px' }}
+        snapAlign="center"
+        ref={lastSnapItem}
```

Comment:
If it does the same, maybe better to do the focus to MyItem.

Reply:
Updated!!